src/components/Table/Table.qa.stories.tsx

```diff
@@ -122,9 +116,8 @@ export function NestedFlexible() {
 // 2. Select individual "Project Items". Note parent is "checked"
 // 3. Remove filter, additional project items under parent are "unchecked", though parent still shows "checked" instead of what it should show as "indeterminate".
 export function Filterable() {
-  // Using a useState so that useComputed works
-  const [api, setApi] = useState<GridTableApi<BeamNestedRow> | null>();
-  const selectedIds = useComputed(() => api?.getSelectedRows().map((r) => r.id) || [], [api]);
+  const api = useGridTableApi<BeamNestedRow>();
+  const selectedIds = useComputed(() => api.getSelectedRows().map((r) => r.id), [api]);
```

Comment:
This is the usage this approach unlocks: the page can immediately bind a `useComputed` against the `api` and not worry about "api is not available yet" / "I need to trigger a re-render just to see it's set value".